diff --git a/node_build/make.js b/node_build/make.js
index 83a311d..550f625 100644
--- a/node_build/make.js
+++ b/node_build/make.js
@@ -252,6 +252,9 @@ Builder.configure({
         builder.config.includeDirs.push(
             BUILDDIR+'/dependencies/libuv/include/'
         );
+        builder.config.cflags.push(
+            '-fno-stack-protector'
+        );
         var libuvBuilt;
         var python;
         nThen(function (waitFor) {
